### ISSUE TITLE
Multiple reference pointer for content (within a page)

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SemanticInterlanguageLinks#8]
@kghbln Speaking from a conceptional point (not from a technical) can it ever happen that a page `A` (with assigned language `en` and a reference point `Foo`) representing some content would need to point to different locations (such as A with `en`, `Bar`) for the same content and language?

```
          linked by `Bar`
     |                      |
C ( `ja`, `Bar`)   A ( `en`, `Bar`)
                   A ( `en`, `Foo`)   B (`ja`, `Foo`)
                      |                    |
                         linked by `Foo`
```

I can't really find a use case that would support such scenario since the representation of the content would be different in case of a different reference pointer but maybe I missed something.

PS: Above scenario is currently not supported. Redirects, or aliases (owl:sameAs) are different concepts and therefore are not part of the discussion.

**Comment by kghbln** [comment]
If there is a page "Bar and Foo" in one language whereas other languages have separate pages for "Bar" and "Foo". On smw.o "Bar" could be "Help:Properties" and "Foo" "Help:Types" whereas an artifact "Help:Properties and types" is still around in a different language. This is an edge case since that page should be split as well at some point. For the transition period such a reference could be helpful. Other cases do not come to my mind right now.

**Comment by mwjames** [comment]
Added/fixed with 9dc8dfdc25.

A user has to be careful to avoid circular references when s(he) uses more than one `interlanguagelink` parser call from the same subject.
